log_merger.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+import pandas as pd
+
+
+class LogMerger:
+    def __init__(self,
+                 infile: str,
+                 msg_types: list[str],
+                 max_msgs: int,
+                 max_rows: int,
+                 verbose: bool):
+        self.infile = infile
+        self.prefix = infile.split('.')[0]
```

Comment:
Are you using line 14 because self.infile may be something like '/dir1/dir2/file.tar.gz' and you only want '/dir1/dir2/file'? Otherwise the following will be more consistent with the outfile method on line 21.
```
self.prefix, _ = os.path.splitext(self.infile)
```

Reply:
Ah, this is dead code! I'll remove it.